tests/bud/copy-from-keep-ownership/Dockerfile

```diff
@@ -0,0 +1,5 @@
+FROM busybox AS basis
+RUN echo hello > /newfile && chown 123:456 /newfile
+FROM basis
+COPY --keep-ownership --from=basis /newfile /newfile_123_456
```

Comment:
The `--keep-ownership` option is redundant when there's a `--from`, so I don't know that we need this test.

Reply:
you are right